tck/param/sdk.go

```diff
@@ -0,0 +1,9 @@
+package param
+
+type SetupParams struct {
```

Comment:
Is there are reason for some of these being pointers and others not?

Reply:
Basically the ones that are being pointers are optional -> they can be nil.

Reply:
I will try to make every param struct be like this.

Reply:
Done